hs/src/Reach/Simulator/Core.hs

```diff
@@ -31,34 +29,48 @@ data LocalState = LocalState
 type ActorId = Int
 type PoolId = Integer
 type AccountId = Int
-type Locals = M.Map ActorId LocalState
+type Locals = M.Map ActorId LocalInfo
 
-data State = State
+data Global = Global
   { e_record :: [Transmission]
   , e_ledger :: Ledger
   , e_linstate :: LinearState
   , e_nwtime :: Integer
   , e_nwsecs :: Integer
-  , e_locals :: Locals
-  , e_curr_actorid :: ActorId
-  , e_nactid :: ActorId
+  , e_nactorid :: ActorId
   , e_naccid :: AccountId
   , e_nmsgid :: PoolId
+  , e_partacts :: M.Map Participant ActorId
   }
 
+data Local = Local
+  { e_locals :: Locals
```

Comment:
Quibble: I don't understand why this isn't `LocalInfo`. In other words, I think that `Locals` is something that only lives in `Server.hs` not `Core.hs`

Reply:
yeah

Reply:
actually i'd rather not do this right now because: the Locals in the Server.hs could become out of sync with the LocalInfo in Core.hs if i do